foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/pipeline.driver.py.vm

```diff
@@ -1,10 +1,6 @@
 #foreach($step in $steps)
-#if ($step.isGeneric())
-from ${packageFolderName}.impl.${step.getLowercaseSnakeCaseName()} import ${step.capitalizedName}
```

Comment:
I made the change in this ticket from `impl` to `step` directory so that it reflects the pyspark directory of having steps in `step`. If this is a breaking change that doesn't fit too well for this ticket, I can revert it.

Reply:
Honestly I think it's fine.  Our [docs](https://boozallen.github.io/aissemble/aissemble/current/pipeline-metamodel.html#_pipeline_step_options) don't even list `generic` as a supported step type for data pipelines, so AFAIK it's an unreleased feature.  I think there could be reasonable debate on whether it's a worthwhile feature as well. So I think simplifying our templates with a breaking change as you've done here is perfectly fine. (And probably preferable.)